README.md

```diff
@@ -1,8 +1,11 @@
-# bevy_editor_prototypes
+# Bevy Editor Prototypes
 
-Low-friction experiments for the bevy_editor.
+> [!WARNING]
+> Bevy Editor is still a prototype in development and testing. 
+> Everything you see is still to be working or cleaned up. read about what we're building in the [Design Book](https://bevyengine.github.io/bevy_editor_prototypes/)! or [Figma document]
+> to run just type this ```cargo run``` or ```cargo run --example simple_editor``` command in the terminal of the ```bevy_editor_prototypes folder```
 
-Read about what we're building in the [Design Book](https://bevyengine.github.io/bevy_editor_prototypes/)!
+Bevy Editor Prototypes will be the next editor for [Bevy Engine](https://bevyengine.org/). is still being tested, help us to contribute!
```

Comment:
```suggestion
```
I don't think that this line is helpful, and I think it implies a stronger endorsement than we'd like to give at the current stage :)